iceoryx_posh/include/iceoryx_posh/internal/mepoo/shared_chunk.hpp

```diff
@@ -35,7 +35,7 @@ class SharedChunk
   public:
     SharedChunk() = default;
     SharedChunk(ChunkManagement* const f_resource);
-    SharedChunk(const relative_ptr<ChunkManagement>& f_resource);
+    SharedChunk(const rp::RelativePointer<ChunkManagement>& f_resource);
```

Comment:
I liked the STL syntax more. Can stay like this but we should consider using snake case in utils. It also got more verbose for no good reason, always a negative in my book.